lua-libraries/lunamark/reader/markdown.lua

```diff
@@ -1661,6 +1665,33 @@ function M.new(writer, options)
                   * larsers.table_caption^-1
                   / writer.table
 
+  ------------------------------------------------------------------------------
+  -- Math
+  ------------------------------------------------------------------------------
+
+  -- Closing $ must have a non-space character immediately to its left,
+  --       and must not be followed immediately by a digit.
+  -- NOTE: For display math, "there can be no blank lines between the opening and closing $$ delimiters",
+  -- Does this implies blank lines are acceptable in inline math? For now, assume it's not the case,
+  -- and not even a line break (which would be allowed in $$ per the above)
```

Comment:
Unless I got it wrong, Github doesn't accept a line break in inline math... So `$e^{i\pi}$` is $e^{i\pi}$, but
```
$e^{i\pi
= -1$
```
Is not parsed as math:
$e^{i\pi}
 = -1$
Pandoc seems to differ here: accepts a line break, but seems to have the same restriction as display math (= no blank lines). Really, folks?